scripts/jsonnetfile.json

```diff
@@ -0,0 +1,13 @@
+{
+    "dependencies": [
+        {
+            "name": "kube-state-metrics",
+            "source": {
+                "local": {
+                    "directory": "../jsonnet/kube-state-metrics"
+                }
+            },
+            "version": ""
```

Comment:
Should we pin this to master?

Reply:
it's a directory based package so it will always do whatever is currently in the directory